CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v0.15.1
+
+### Added
+
+- Update metallb to `v0.12.1`.
+
+### Fixed
```

Comment:
If you intend to only add that change to changelog I'd:

- drop the `Fixed` section
- change `Added` to `Improvements` as this change doesn't really add anything new.


```suggestion
### Improvements

- Update metallb to `v0.12.1`.
```

Reply:
Sounds good.